data/initiatives.yml

```diff
@@ -12,7 +12,7 @@
   link: "https://ccss.carleton.ca/"
   color: "#ffab40"
   description: "Interactive content for COMP 1805 and COMP 2804. (Coming Fall 2021)"
```

Comment:
Could you also update the description to no longer say "coming fall 2021"

Reply:
Committed the change that removes that message